strawberry/django/views.py

```diff
@@ -231,7 +231,8 @@ def as_view(cls, **initkwargs: Any) -> Callable[..., HttpResponse]:
         # https://docs.djangoproject.com/en/3.1/topics/async/#async-views
 
         view = super().as_view(**initkwargs)
-        view._is_coroutine = asyncio.coroutines._is_coroutine  # type: ignore[attr-defined]
+        markcoroutinefunction(view)
```

Comment:
do we need to add tests for this?

Reply:
Hrm, TBH it would be nice to have one

I was just trying to fix this with minimum effort, specially since strawberry-graphql-django tests this extensively =P, but I'll try to add one here :)